hellosign_sdk/utils/request.py

```diff
@@ -166,9 +166,31 @@ def post(self, url, data=None, files=None, headers=None, get_json=True):
 
         response = requests.post(url, headers=self.headers, data=data, auth=self.auth, files=files, verify=self.verify_ssl)
         json_response = self._process_json_response(response)
-        
+
         return json_response if get_json is True else response
 
+    def delete(self, url, headers=None):
```

Comment:
delete what?

Reply:
this one's just adding a generic `delete` function to be able to make these kinds of requests since this SDK's only been able to make get and post requests before

Reply:
Sounds good, would it be worth adding some guidance in comments about what object types can be deleted.

Reply:
good call! i added some detail to the comments